Core/Utilities/ElasticSearchDB.py

```diff
@@ -23,6 +24,9 @@
 from DIRAC.FrameworkSystem.Client.BundleDeliveryClient import BundleDeliveryClient
 
 
+LOG = gLogger.getSubLogger(__name__)
```

Comment:
We use all-capitals names for constants, not for objects. I think the name should be changed

Reply:
But its a constant object... and this was used also in other places.

Reply:
I mean this convention is for string or int/float constants not for objects. Constant object is a strange concept to me

Reply:
I don't mind either option, but I point out that this same line is also in use elsewhere.

Reply:
I would then change everywhere to subLogger or gSubLogger. Why are we defining conventions which we are diluting easily ?

Reply:
I have changed it to `sLog` (in several places)

Reply:
What convention ? There's nothing in there as far as I can tell https://dirac.readthedocs.io/en/latest/DeveloperGuide/CodingConvention/index.html
And if you go that way, you give an argument why it should be `LOG`:
```
>>> class A(object):
...   pass
... 
>>> type(A) == type(int)
True
```